doc/read-the-docs-site/tutorials/AuctionValidator.hs

```diff
@@ -9,7 +8,7 @@ import PlutusCore.Default qualified as PLC
 import PlutusLedgerApi.V1 (POSIXTime, PubKeyHash, Value, adaSymbol, adaToken, singleton)
 import PlutusLedgerApi.V1.Address (pubKeyHashAddress)
 import PlutusLedgerApi.V1.Interval (contains)
-import PlutusLedgerApi.V2 (Datum (..), OutputDatum (..), ScriptContext (..), TxInfo (..), TxOut (..), from, to)
```

Comment:
I don't think this is too long? It just needs to be split over multiple lines.

Reply:
I think there's an ambiguity: I don't think the intention was that "long import statement" meant "the import statement hits the line length", but rather "the import statement is annoyingly long".

Reply:
I edited our stylish-haskell config. It seems the "column" config only applies to imports. Also, for those who cases, we can configure how the imports that are too long are broken up: https://github.com/haskell/stylish-haskell/blob/13f1db77f28b62ac6da6abb3c82244d5e740503b/data/stylish-haskell.yaml#L198